src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -605,31 +607,44 @@ public void setHomeBlock(@Nullable TownBlock homeBlock) {
 			setWorld(homeBlock.getWorld());
 
 		// Unset the spawn if it is not inside of the new homeblock.
-		if (spawn != null && !homeBlock.getWorldCoord().equals(WorldCoord.parseCoord(spawn))) {
+		if (spawn != null && !homeBlock.getWorldCoord().equals(Coord.parseCoord(spawn))) {
```

Comment:
can a WorldCoord equals a Coord? It should be good but it would be nice to know this was tested to still work.

Reply:
Works just fine.
<img width="325" alt="image" src="https://user-images.githubusercontent.com/2712890/144542527-4f9fdd18-e57b-4bd4-b1ac-8c7ec78bf916.png">